.evergreen/install-npm-deps.sh

```diff
@@ -1,7 +1,7 @@
 set -e
 set -x
 
-npm ci --verbose
+npm ci --verbose --legacy-peer-deps
```

Comment:
Leaving this comment as a reminder to myself to revert aee60aa5c532e0f3f3ba154aec76639879c83243 later